src/test/java/seedu/address/storage/JsonAdaptedMemberTest.java

```diff
@@ -90,28 +97,17 @@ public void toModelType_invalidEmail_throwsIllegalValueException() {
         assertThrows(IllegalValueException.class, expectedMessage, member::toModelType);
     }
 
-    @Test
-    public void toModelType_nullEmail_throwsIllegalValueException() {
-        JsonAdaptedMember member = new JsonAdaptedMember(VALID_NAME, VALID_PHONE, null,
-                VALID_ADDRESS, VALID_POSITIONS, VALID_TASK_LIST);
-        String expectedMessage = String.format(MISSING_FIELD_MESSAGE_FORMAT, Email.class.getSimpleName());
-        assertThrows(IllegalValueException.class, expectedMessage, member::toModelType);
-    }
-
     @Test
     public void toModelType_invalidAddress_throwsIllegalValueException() {
         JsonAdaptedMember member =
                 new JsonAdaptedMember(VALID_NAME, VALID_PHONE, VALID_EMAIL,
                         INVALID_ADDRESS, VALID_POSITIONS, VALID_TASK_LIST);
         String expectedMessage = Address.MESSAGE_CONSTRAINTS;
-        assertThrows(IllegalValueException.class, expectedMessage, member::toModelType);
-    }
-
-    @Test
-    public void toModelType_nullAddress_throwsIllegalValueException() {
-        JsonAdaptedMember member = new JsonAdaptedMember(VALID_NAME, VALID_PHONE, VALID_EMAIL,
-                null, VALID_POSITIONS, VALID_TASK_LIST);
-        String expectedMessage = String.format(MISSING_FIELD_MESSAGE_FORMAT, Address.class.getSimpleName());
+        try {
```

Comment:
Why is there try catch here haha? Wouldn't the assertthrow below do the same thing? Pls clarify, I'm not sure if this is correct